site/FAQ.php

```diff
@@ -21,36 +21,18 @@
 		<div class="DivideLine">
 		</div>
 		<div class="Questions">
-			<p class="QuestionTitle">
-				What is the question?
-			</p>
-			<p class="QuestionAnswer">
-				This is an answer for the question listed above.
-			</p>
-			<p class="QuestionTitle">
-				What is the question?
-			</p>
-			<p class="QuestionAnswer">
-				This is an answer for the question listed above.
-			</p>
-			<p class="QuestionTitle">
-				What is the question?
-			</p>
-			<p class="QuestionAnswer">
-				This is an answer for the question listed above.
-			</p>
-			<p class="QuestionTitle">
-				What is the question?
-			</p>
-			<p class="QuestionAnswer">
-				This is an answer for the question listed above.
-			</p>
-			<p class="QuestionTitle">
-				What is the question?
-			</p>
-			<p class="QuestionAnswer">
-				This is an answer for the question listed above.
-			</p>
+			<?php
+				$FAQ=array (
```

Comment:
This looks like you've got the right idea in terms of how to make an array and loop through stuff, but why don't you take it to the next level. Eventually we're going to have a function called maybe `GetFaqQuestions()` which will pull from the database and return an array like this.

So how about you make a file called maybe faq.php which you include from init, and then make that a function which currently returns this hardcoded array? That way it'll be easy to replace the array with a database query later on, and this front-end code (the part that generates the html) won't need to be updated at all.

That's the beauty of functions. If you write them in a modular way, the code that uses them doesn't have to care what's happening behind the scenes. So switching from hardcoding arrays to using database queries is contained within a single function, and the rest of the code is blissfully unaware.